backend/src/features/profiles/profiles.service.ts

```diff
@@ -0,0 +1,39 @@
+import { Injectable } from '@nestjs/common';
+import { InjectConnection, InjectModel } from '@nestjs/mongoose';
+import { Connection, Model } from 'mongoose';
+import { COLLECTION_NAME } from 'src/configs/mongoose.config';
```

Comment:
Dont use root path for import.
Please change to relative

Reply:
### same as your Quizzes module

<img width="883" alt="image" src="https://github.com/user-attachments/assets/20df54e6-075f-4692-8afb-799b27533172">

Reply:
Thank you. I will update my component